.circleci/regenerate.py

```diff
@@ -122,15 +122,14 @@ def upload_doc_job(filter_branch):
 
 
 manylinux_images = {
-    "cu102": "pytorch/manylinux-cuda102",
     "cu116": "pytorch/manylinux-cuda116",
     "cu117": "pytorch/manylinux-cuda117",
 }
 
 
 def get_manylinux_image(cu_version):
     if cu_version == "cpu":
-        return "pytorch/manylinux-cuda102"
+        return "pytorch/manylinux-cuda116"
```

Comment:
Why cuda102 is needed here?
```suggestion
        return "pytorch/manylinux-cpu"
```